example-loading-into-r/load_csv_into_r.R

```diff
@@ -0,0 +1,24 @@
+###Example of how to load a .csv file from Tidepool into R:
+
+#Step 1, identify the pathway to the file. You can copy and paste this from your file explorer.  
+#Add the name of the file to the end.  Be sure to double the \\ because a single \ indicates an escape character in R.
+
+pathway<-"C:\\Users\\kybol\\Documents\\Github_Tidepool\\data-analytics\\example-data\\example-from-j-jellyfish.csv"

```

Comment:
@KyleBolduc , please update these paths so that they are relative. in python it looks like this: os.path.join("..", "..","data-analytics", "example-data", example-from-j-jellyfish.csv")